README.adoc

```diff
@@ -19,7 +19,7 @@
 = Reactive client for Apache Pulsar
 
 :github: https://github.com/apache/pulsar-client-reactive
-:latest_version: 0.1.0
+:latest_version: 0.2.0
```

Comment:
I think this should be done separately since we haven't released 0.2.0 yet.

Reply:
ok